application/forms/Command/Object/ToggleObjectFeaturesForm.php

```diff
@@ -47,6 +55,13 @@ public function __construct($featureStatus)
         ];
 
         $this->getAttributes()->add('class', 'object-features');
+
+        $this->on(self::ON_SUCCESS, function () {
+            $status = $this->submittedFeature->getEnabled() ? 'enabled' : 'disabled';
+            $label = $this->features[$this->submittedFeature->getFeature()]['label'];
+
+            Notification::success(sprintf(t('%s %s successfully'), $label, $status));
```

Comment:
* `Enabled active checks successfully`, `Disabled active checks successfully`, ...
* Support for multiple notifications